doc/engineering-reference/src/operational-faults.tex

```diff
@@ -315,3 +318,139 @@ \subsubsection{Modeling Approach}\label{modeling-approach-3}
 \(_{design}\) is for the parameters in the design condition;
 
 \(_{df}\) is for the parameters in the dirty filter case.
+
+
+\subsection{Chiller Supply Water Temperature Sensor Offset}\label{chiller-supply-water-temperature-sensor-offset}
+
+\subsubsection{Symptom}
+
+The chiller supply water temperature readings deviate from the actual water temperature levels due to sensor offset at the evaporator outlet. This can lead to incorrect chiller supply water temperature, and thus the inappropriate and inefficient chiller operations.
+
+\subsubsection{Modeling Approach}
+
+The fault applies to a number of chiller types, namely:
+
+\begin{itemize}
+\tightlist
+\item
+  Chiller:Electric
+\item
+  Chiller:Electric:EIR
+\item
+  Chiller:Electric:ReformulatedEIR
+\item
+  Chiller:ConstantCOP
+\item
+  Chiller:EngineDriven
+\item
+  Chiller:CombustionTurbine
+\item
+  Chiller:Absorption
+\item
+  Chiller:Absorption:Indirect
+\end{itemize}
+
+These chillers can have different flow modes:
+
+\begin{itemize}
+\tightlist
+\item
+  \emph{ConstantFlow} for constant pumping with flow controlled by chiller to operate at full design flow rate.
+\item
+  \emph{LeavingSetpointModulated} for variable pumping with flow controlled by chiller to vary flow to target a leaving temperature setpoint.
+\item
+  \emph{NotModulated} for either variable or constant pumping with flow controlled by the external plant system.
+\end{itemize}
+
+\newline \newline
+
+\textbf{(a) constant flow chillers}
+
+For the chillers with \emph{ConstantFlow} and \emph{NotModulated}, local control is provided by resetting the leaving water temperature. The actual evaporator outlet water temperature value at faulty operations can be obtained via:
+
+\begin{equation}
+T_{evap-o,f} = T_{evap-o,ff} - \Delta T
+\end{equation}
+
+Where, 
+
+\(T_{evap-o,f}\) evaporator outlet temperature in the faulty case (actual value)
+
+\(T_{evap-o,ff}\) evaporator outlet temperature in the fault-free case (reading value)
+
+\(\Delta T\) difference between the temperature reading and the actual temperature
+
+Then the evaporator capacity can be calculated with:
+
+\begin{equation}
+Q_{evap,f} = m_{evap} \times C_p \times (T_{evap-i} - T_{evap-o,f} )
+\end{equation}
+
+Where, 
+
+\(m_{evap}\) evaporator water flow rate (design value, actual value)
+
+\(Q_{evap,f}\) actual evaporator capacity in the faulty case
+
+\(T_{evap-o,f}\) evaporator outlet temperature value in the faulty case (actual value)
+
+\(T_{evap-i}\) evaporator inlet temperature value 
+
+\newline \newline
```

Comment:
@mjwitte I've replaced "\newline" with "\leavevmode \\". I think this can solve the problem.

I've never built the whole engineering ref locally. So can you please test it for me? Very appreciate it.